conda_smithy/tests/test_lint_recipe.py

```diff
@@ -206,6 +206,38 @@ def test_redundant_license(self):
                             'the word "License".')
         self.assertIn(expected_message, lints)
 
+    def test_end_empty_line(self):
+        bad_contents = [
+            # No empty lines at the end of the file
+            b'extra:\n  recipe-maintainers:\n    - goanpeca',
+            b'extra:\r  recipe-maintainers:\r    - goanpeca',
+            b'extra:\r\n  recipe-maintainers:\r\n    - goanpeca',
+            # Two empty lines at the end of the file
+            b'extra:\n  recipe-maintainers:\n    - goanpeca\n\n',
+            b'extra:\r  recipe-maintainers:\r    - goanpeca\r\r',
+            b'extra:\r\n  recipe-maintainers:\r\n    - goanpeca\r\n\r\n',
+            # Three empty lines at the end of the file
+            b'extra:\n  recipe-maintainers:\n    - goanpeca\n\n\n',
+            b'extra:\r  recipe-maintainers:\r    - goanpeca\r\r\r',
+            b'extra:\r\n  recipe-maintainers:\r\n    - goanpeca\r\n\r\n\r\n',
+        ]
```

Comment:
Nope. Looks good. Thanks.

Adding `.gitattributes` sounds like a good idea. Maybe we should write up an issue. Alternatively, if you would like to lead the way with a PR, that would be welcome too.